### PR TITLE
Releasing v0.9.12 - replacing XPath library

### PR DESCRIPTION
**Pull request by SeanCondon**
Changing the XPath lib to an extended one with the new `set-contains` function

### CODE REVIEW

**Comment by SeanCondon** [comment]
retest this please

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/onosproject/config-models
 go 1.16
 
 require (
-	github.com/antchfx/xpath v1.2.0
+	github.com/SeanCondon/xpath v0.0.0-20220217125907-c2b75876708f
```
Comment (teone) [inline go.mod:6]: is this to fix a bug in the upstream xpath?

**Review by teone — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (SeanCondon) [reply go.mod:6]: I needed to add a new function to the upstream XPath

Comment (SeanCondon) [reply go.mod:6]: I doubt they'd take it because I have added a new function "set-contains" that's not in the XPath standard

**Pull request merged**